test/test_write.jl

```diff
@@ -0,0 +1,57 @@
+# Test writing component of SegyIO
+global key = ["GroupX";"GroupY";"SourceX";"SourceY"]
+global dir = string(SegyIO.myRoot,"data/")
```

Comment:
`string(SegyIO.myRoot,"/data/")` need the slash

Reply:
> `string(SegyIO.myRoot,"/data/")` need the slash

My bad, I just modified it by using "../data". Hope it will work this time.

Reply:
Use joinpath to combine path elements, forget about slashes.

Reply:
> Use joinpath to combine path elements, forget about slashes.

Thanks for the comment, Henryk. It's good to know the joinpath.